px-vis-xy-chart.html

```diff
@@ -851,6 +892,39 @@
 
       }, 5);
     },
+    _calcMarkerMargins: function() {
+      if(this.hasUndefinedArguments(arguments)) {
+        return;
+      }
+
```

Comment:
Can you please cleanup the whitespace here? There should be no more than one consecutive newline between code blocks.

Reply:
Done: Updated changes in new commit.